modules/tinymce/src/themes/silver/main/ts/ui/core/ChoiceControls.ts

```diff
@@ -26,6 +26,9 @@ interface ControlSpec<T> {
 
   readonly getCurrent: (editor: Editor) => Optional<T>;
   readonly setCurrent: (editor: Editor, value: T) => void;
+
+  readonly onToolbarSetup?: (api: Toolbar.ToolbarMenuButtonInstanceApi) => () => void;
```

Comment:
These probably shouldn't be optional. This sort of functionality seems like something that any ChoiceControl should have (and kind of looks like a line-height bug at the moment).

Reply:
Line height probably should not have that... as line height is much closer to a "bespoke" select and none of those show as active. To be honest I'd prefer if this didn't have it either, but it's required for backwards compatibility.